.gitignore

```diff
@@ -1,32 +1,253 @@
-*.acn
```

Comment:
Why all the extra extensions here? Are these file types now generated by a typical compilation?

Reply:
Not by a _typical_ compilation, no. By _any_ compilation maybe, it is pretty much a catchall ignore file for tex projects. I pulled that in from @berquist and think it is a good addition.